packages/plugin-dom-helpers/src/DomHelpers.ts

```diff
@@ -175,6 +175,32 @@ export class DomHelpers<T extends JWPluginConfig = JWPluginConfig> extends JWPlu
         }
         await this.editor.dispatcher.dispatchHooks('@redraw');
     }
+    /**
+     * Prepend a DOM Node to another DOM Node's children.
+     *
+     * @param params
+     */
+    async prepend(fromDomNode: Node, toDomNode: Node): Promise<void> {
```

Comment:
I don't understand what "prepend from" and "prepend to" mean. If I prepend "to a node" then I prepend inside this node ? Then `toDomNode` is the node in which the node is prepended ? Or I prepend "from" the node ? I have no idea what that means as I think this is not proper english. Obviously, `append` has the same problem. I suggest `node` and `container` for parameters. cc @Zinston if I'm just dumb by not understanding the english of "from" and "to" as prepositions for "prepend".

Reply:
Corrected